src/pages/Debug/Report/DebugReportActions.tsx

```diff
@@ -20,38 +19,42 @@ type DebugReportActionsProps = {
 function DebugReportActions({reportID}: DebugReportActionsProps) {
     const {translate, datetimeToCalendarTime} = useLocalize();
     const styles = useThemeStyles();
+    const [searchValue, setSearchValue] = useState('');
```

Comment:
```suggestion
    const [searchValue, debouncedSearchValue, setSearchValue] = useDebouncedState('');
```

Let's use `useDebouncedState` for this state since we need to filter/map all report actions in each search value change. See example of usage [here](https://github.com/Expensify/App/blob/main/src/components/CategoryPicker.tsx).

Reply:
Good point, updated!